pelican/tests/output/basic/a-markdown-powered-article.html

```diff
@@ -1,65 +1,68 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+    <head>
         <meta charset="utf-8" />
         <title>A markdown powered article</title>
         <link rel="stylesheet" href="/theme/css/main.css" />
         <link href="/feeds/all.atom.xml" type="application/atom+xml" rel="alternate" title="A Pelican Blog Atom Feed" />
-</head>
+    </head>
 
-<body id="index" class="home">
+    <body id="index" class="home">
         <header id="banner" class="body">
-                <h1><a href="/">A Pelican Blog </a></h1>
-                <nav><ul>
-                    <li><a href="/tag/oh.html">Oh Oh Oh</a></li>
-                    <li><a href="/override/">Override url/save_as</a></li>
-                    <li><a href="/pages/this-is-a-test-page.html">This is a test page</a></li>
-                    <li><a href="/category/bar.html">bar</a></li>
-                    <li class="active"><a href="/category/cat1.html">cat1</a></li>
-                    <li><a href="/category/misc.html">misc</a></li>
-                    <li><a href="/category/yeah.html">yeah</a></li>
-                </ul></nav>
+            <h1><a href="/">A Pelican Blog </a></h1>
+            <nav>
+                <ul>
+                            <li><a href="/tag/oh.html">Oh Oh Oh</a></li>
```

Comment:
```suggestion
                        <li><a href="/tag/oh.html">Oh Oh Oh</a></li>
```

If we're going for an OCD-correct indentation, this list should go four spaces back I think ;)